src/ibek/ioc_cmds/commands.py

```diff
@@ -30,7 +30,8 @@ def build_docker(
             help="The filepath to the Dockerfile to build",
             autocompletion=lambda: [],  # Forces path autocompletion
         ),
-    ] = Path.cwd() / "Dockerfile",
+    ] = Path.cwd()
+    / "Dockerfile",
```

Comment:
Is this intentional? Ruff seems to be happy with it on oneline. I think that looks better.

Reply:
I hit ctrl-S and it reformatted as you suggest. Not sure why that was like that.

Reply:
ah but then black complained about it and failed the CI.

Reply:
no not black - ruff - must be a mismatch between versions of ruff somewhere?

Reply:
CI is good now so leaving this for now.